docs/users/extend/custom-compose-files.md

```diff
@@ -1,17 +1,22 @@
 <h1>Defining an additional service with Docker Compose</h1>
 
 ## Prerequisite knowledge
+
 The majority of ddev's customization ability and extensibility comes from leveraging features and functionality provided by [Docker](https://docs.docker.com/) and [Docker Compose](https://docs.docker.com/compose/overview/). Some working knowledge of these tools is required in order to customize or extend the environment ddev provides.
 
 ## Background
-Under the hood, ddev uses docker-compose to define and run the multiple containers that make up the local environment for a web application. Docker Compose supports defining multiple compose files to facilitate [sharing Compose configurations between files and projects](https://docs.docker.com/compose/extends/), and ddev is designed to leverage this ability. Additional services can be added to your project by creating additional docker-compose files in the `.ddev` directory for your project. Any files using the `docker-compose.[servicename].yml` naming convention will be automatically processed by ddev to include them in executing docker-compose functionality. A `docker-compose.override.yml` can additionally be created to override any configurations from the main docker-compose file or any service compose files added to your project.
+
+Under the hood, ddev uses docker-compose to define and run the multiple containers that make up the local environment for a web application. Docker Compose supports defining multiple compose files to facilitate [sharing Compose configurations between files and projects](https://docs.docker.com/compose/extends/), and ddev is designed to leverage this ability.
+
+To add services to your project, create docker-compose files in the `.ddev` directory for your project. ddev will process any files using the `docker-compose.[servicename].yml` naming convention and include them in executing docker-compose functionality. In addition, create a `docker-compose.override.yml` to override any configurations from the main docker-compose file or any service compose files added to your project.
 
 ## Conventions for defining additional services
-When defining additional services for your project, it is recommended to follow these conventions to ensure your service is handled by ddev the same way the default services are.
 
-- Containers should be follow the naming convention `ddev-[projectname]-[servicename]`
+When defining additional services for your project, we recommended you follow these conventions to ensure ddev handles your service the same way ddev handles default services.
```

Comment:
"we recommend you follow". (present tense)

But how about "We recommend these conventions"?